sycl/doc/extensions/README.md

```diff
@@ -1,3 +1,34 @@
 # Extensions
 
-This is where documents can be found that propose extensions to the SYCL specification.
+The directory contains documents that describe DPC++ extensions to SYCL
+specification.
+
+DPC++ extensions status:
```

Comment:
Could we add a legend with possible "Status" values?

Reply:
I have a selfish interest to capture information about the compiler options to turn on/off these extensions.
Do you think we can add another column?

Reply:
A column with compiler options which turn on/off specific extensions?
What do you think about having them in https://github.com/intel/llvm/blob/sycl/sycl/doc/UsersManual.md
and add a link from this doc to the user manual?

Reply:
Sounds good to me.

AFAIK, some extensions are "library-only" and there is no dedicated clang driver option to manage library extensions - they are managed through macro definitions.

Reply:
>  they are managed through macro definitions.

Can we have such macro documented in the user manual as well?

Reply:
I think we should document them and align extensions API with this proposal: https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/ExtensionMechanism/SYCL_INTEL_extension_api.asciidoc.

Reply:
Added a link to user manual, please, let me know if something else should be done in scope of this comment.